packages/types/index.ts

```diff
@@ -17,7 +17,7 @@ export * from './ContactJob';
 export * from './Files';
 export * from './ResourcesImport';
 export * from './Resources';
-export * from './Results';
+export * as Result from './Result';
```

Comment:
I would suggest that we stick to the same pattern as other types for now. If a user needs/wants the entire module for some reason they can do `import * as R from '@tech-matters/types/Result';` If they just want specific parts, they can use the basic import. If we change the pattern here, I think we should change it everywhere which is a larger conversation.